tasks/xgettext.js

```diff
@@ -228,14 +242,15 @@ module.exports = function(grunt) {
         extractDirectiveStrings('"', func);
       });
 
-      return messages;
+      return [messages, messageToFilesMap];
```

Comment:
Consider returning an object rather than array so there isn't confusion over ordering, here and elsewhere.